docs/pages/components/object-marker.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Object Marker
+id:
+keywords: object Marker, Marker, object, object-Marker, clickable
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/object-Marker.html
+folder: components
+tags: [f3, a11y, theme]
+---
+Object marker indicates the technical status of an object. It display the technical state like (draft, unsaved changes, locked, favorite, flagged). Use the object marker for this unless you want to display the status of the object in the business life cycle. The technical status can be represented as an icon, with an icon and text, or as text only, depending on the screen size.
+{: .docs-intro}
+<br>
+
+# Object Marker Icon Only
+{% capture badge %}
+<span class="fd-object-marker sap-icon--request"></span>
+<span class="fd-object-marker sap-icon--add-favorite"></span>
+<span class="fd-object-marker sap-icon--flag"></span>
+<span class="fd-object-marker sap-icon--user-edit"></span>
+<span class="fd-object-marker sap-icon--private"></span>
```

Comment:
If these are being used to indicate some sort of status they will need `aria-label`s.